main.py

```diff
@@ -113,6 +113,15 @@ def get(self):
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
 
+    Args:
+        regex_route: str. For regex for routing
+        handler: function. Handler function for the admin page.
+        defaults: dict. Will be passed in the 
+            redirect if there is any.
+
+    Returns:
+        routes.RedirectRoute: Return redirect route module
```

Comment:
Move these lines after Warnings and change the description of return `RedirectRoute. The redirect route object.`

Reply:
done

Reply:
I don't see any changes here*